docs/source/how_to/use_oauth2.rst

```diff
@@ -46,13 +46,13 @@ request with the following parameters.
 Parameter                  Description
 ========================= =====================================================================
 **client_id** (required)   The client ID you obtained from the dashboard
-**redirect_url**           One of the redirect URLs you have registered on the dashboard
+**redirect_uri**           One of the redirect URLs you have registered on the dashboard
```

Comment:
I think `URI` makes sense because one can always encode the special characters and it still works. I think `oauth2_provider` decodes it. For eg, redirecting to http%3A%2F%2Flocalhost%3A4000%2Fonlogin also works

So maybe we can change "One of the redirect URLs you have registered on the dashboard" to "One of the redirect URIs you have registered on the dashboard"?

Reply:
I agree, just make it URI across the page, code and text and all.

Reply:
I have changed all the textual references from URL to URI. I have made the corresponding changes on the developers' dashboard to eliminate any confusion that may arise due to the differences from the documentation.